src/plugins_frame.rs

```diff
@@ -274,3 +314,13 @@ pub async fn perform_install(mut plugin: EspimPlugin) -> EspimPlugin {
     }
     plugin
 }
+
+fn validate_url(url: &str) -> bool {
+    match Url::parse(&url) {
+        Ok(parsed_url) => {
+            // Überprüft, ob das Schema http oder https ist
+            parsed_url.scheme() == "http" || parsed_url.scheme() == "https"
+        },
+        Err(_) => false,
+    }
+}
```

Comment:
Not sure in which case the homepage would ever be invalid. The [RFC](https://github.com/endless-sky/rfcs/blob/main/rfcs/0001-plugin-index.md) doesnt explicitly say so (because i didnt consider anyone using non-http), but homepage is very much for http/https only.